### PR TITLE
OCPEDGE-1129: fix: limiting the parallel test count for ibu

### PR DESCRIPTION
**Pull request by jeff-roche**
I added an update to limit the count of concurrent tests. I think this count needs to be corecount - 1 so that the api server doesn't get starved.

### CODE REVIEW

**Comment by jeff-roche** [comment]
/pj-rehearse periodic-ci-openshift-kni-lifecycle-agent-main-nightly-4.16-image-based-install periodic-ci-openshift-kni-lifecycle-agent-main-nightly-4.15-image-based-upgrade-e2e-parallel-conformance periodic-ci-openshift-kni-lifecycle-agent-main-nightly-4.16-image-based-upgrade-e2e-serial-conformance periodic-ci-openshift-kni-lifecycle-agent-main-nightly-4.16-image-based-upgrade-e2e-parallel-conformance periodic-ci-openshift-kni-lifecycle-agent-main-nightly-4.15-image-based-upgrade-e2e-serial-conformance pull-ci-openshift-kni-lifecycle-agent-main-ibu-e2e-flow

**Comment by jeff-roche** [comment]
/pj-rehearse periodic-ci-openshift-kni-lifecycle-agent-main-nightly-4.16-image-based-install periodic-ci-openshift-kni-lifecycle-agent-main-nightly-4.15-image-based-upgrade-e2e-parallel-conformance periodic-ci-openshift-kni-lifecycle-agent-main-nightly-4.16-image-based-upgrade-e2e-serial-conformance periodic-ci-openshift-kni-lifecycle-agent-main-nightly-4.16-image-based-upgrade-e2e-parallel-conformance periodic-ci-openshift-kni-lifecycle-agent-main-nightly-4.15-image-based-upgrade-e2e-serial-conformance pull-ci-openshift-kni-lifecycle-agent-main-ibu-e2e-flow

**Comment by jeff-roche** [comment]
/pj-rehearse periodic-ci-openshift-kni-lifecycle-agent-main-nightly-4.15-image-based-upgrade-e2e-parallel-conformance periodic-ci-openshift-kni-lifecycle-agent-main-nightly-4.16-image-based-upgrade-e2e-serial-conformance periodic-ci-openshift-kni-lifecycle-agent-main-nightly-4.16-image-based-upgrade-e2e-parallel-conformance periodic-ci-openshift-kni-lifecycle-agent-main-nightly-4.15-image-based-upgrade-e2e-serial-conformance

File: ci-operator/step-registry/openshift/image-based/upgrade/e2e/conf/openshift-image-based-upgrade-e2e-conf-commands.sh [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+echo "Waiting for operators to update after registry update"
+sleep 2m
+
+# Loop until all operators are healthy
+until [ -z "\$(oc get co -o json | jq -r '.items[] | select(.status.conditions[] | select(.type=="Available" and .status!="True")) | .metadata.name')" ]; do
+  echo "Operators still rolling out. Checking again in 1 minute"
+  # Wait before checking again
+  sleep 1m
+done
+echo "All operators are healthy."
```
Comment (jakobmoellerdev) [inline ci-operator/step-registry/openshift/image-based/upgrade/e2e/conf/openshift-image-based-upgrade-e2e-conf-commands.sh:51]: maybe what we really want is https://docs.openshift.com/container-platform/4.14/cli_reference/openshift_cli/administrator-cli-commands.html#oc-adm-wait-for-stable-cluster ? it feels like a hardcoded sleep is doomed to fail

File: ci-operator/step-registry/openshift/image-based/upgrade/e2e/conf/OWNERS [hunk 2]
```diff
@@ -0,0 +1,5 @@
+approvers:
```
Comment (jakobmoellerdev) [inline ci-operator/step-registry/openshift/image-based/upgrade/e2e/conf/OWNERS:1]: I saw in some other folders we symlink to the parent folder, should we do that as well?

**Review by jakobmoellerdev — COMMENTED** [review]
just 2 nits, other than that lgtm

File: ci-operator/step-registry/openshift/image-based/upgrade/e2e/conf/OWNERS (hunk 2 above)
Comment (jeff-roche) [reply ci-operator/step-registry/openshift/image-based/upgrade/e2e/conf/OWNERS:1]: I've been thinking of converting all of the ibu/ibi stuff to that approach, I'll open a separate PR with that

**Pull request merged**